.github/workflows/check.yml

```diff
@@ -46,6 +46,9 @@ jobs:
     - uses: actions/setup-python@v2
       with:
         python-version: 3.9
+    - uses: actions/setup-python@v2
+      with:
+        python-version: "3.10"
 
     # Check that the scripts work.
-    - run: nox -s check
+    - run: nox -s check-2.7 check-3.5 check-3.6 check-3.7 check-3.8 check-3.9 check-3.10
```

Comment:
Nox defaults to `--no-error-on-missing-interpreters` so it's not necessary to repeat the list of interpreters we install.

Reply:
That's odd, because https://github.com/pypa/get-pip/runs/8170133734?check_suite_focus=true shows the CI failing on the missing interpreters.

Reply:
OTOH, https://github.com/pypa/get-pip/runs/7659032802?check_suite_focus=true#step:11:29 shows that this works as I've described. There's likely something else happening with your PR.

Reply:
Bah, they changed behaviours in a release between these two runs.

> Missing interpreters now error the session on CI by default (#567)